source/includes/steps-enable-authentication-in-shardcluster-nodowntime-transition-config.yaml

```diff
@@ -0,0 +1,95 @@
+stepnum: 1
+level: 4
+title: "Add ``transitionToAuth: true`` to the ``mongod`` configuration file."
```

Comment:
Update the mongod configuration file to include authentication settings.

(otherwise, you'd have to mention the keyfile as well).